driver/git.go

```diff
@@ -138,6 +139,14 @@ func (driver *GitDriver) Check(cursor *semver.Version) ([]semver.Version, error)
 
 func (driver *GitDriver) setUpRepo() error {
 	_, err := os.Stat(gitRepoDir)
+	if driver.SkipSSLVerification {
```

Comment:
Seems like a weird place to do this. I think it should be moved into a separate function before the loop that sets up the repo.

Reply:
Ok, I added it before the loops.